### DISCUSSION TITLE
Fail2ban fails on Debian 12

### DISCUSSION THREAD

**Discussion by onelinergeek** [fail2ban/fail2ban#3637] [General]
Fail2ban fails on Debian 12 because it has no /var/log/auth.log file anymore.

**Comment by sebres** [comment]
Shortly, it depends (default log settings, syslog forwarding rules, etc).
Anyway to configure the jail(s), either set proper `logpath` or for journal set `backend` to `systemd`. See #3551, #3417 and others.

**Reply by onelinergeek** [reply]
Thank you so much Sebres. It's working now. I did.
1. Edit comment #logpath, backend = systemd, enabled = true
2. Restart fail2ban service and ssh

**Reply by sebres** [reply]
> Restart fail2ban service **and ssh**

By the way, one doesn't need to restart sshd if one changes fail2ban's config, restart of fail2ban were fully enough.
